LocalPlanner: 
  verbose: true
  visualize_topics: true
  odom_topic: "/odom"
  footprint_topic: "/squirrel_footprint_observer/footprint"
  goal_lin_tolerance: 0.05
  goal_ang_tolerance: 0.05
  max_safe_lin_velocity: 0.7
  max_safe_ang_velocity: 0.9
  max_safe_lin_displacement: 0.5
  max_safe_ang_displacement: 1.0
  collision_based_replanning: true
  replanning_lin_lookahead: 1.0
  replanning_ang_lookahead: 1.0
  replanning_path_length_ratio: 0.75
  safety_observers: ["ScanSafetyObserver", "ArmSkinObserver"]

  LinearMotionPlanner:
    max_linear_velocity: 0.8
    max_angular_velocity: 1.0
    linear_smoother: 0.75
    angular_smoother: 0.75
    time_scaler: 2.0
    waypoints_heading_lookahead: 15
    lookahead: 0.25

  ControllerPID:
    visualize_topics: false
    kP_lin: 1.5
    kP_ang: 1.5
    kI_lin: 0.0001
    kI_ang: 0.0001
    kD_lin: 0.0001
    kD_ang: 0.0001
  
  ScanSafetyObserver:
    verbose: false
    scan_topic: "/scan"
    max_safety_rangevel: 0.5
    unsafe_range: 0.75
    ab_filter: {alpha: 0.1, beta: 0.005}
    enabled: false

  ArmSkinObserver:
    verbose: false
    topic_name: "/arm_skin"
    enabled: false